www/js/insert-slide.js

```diff
@@ -0,0 +1,188 @@
+const sanitizeHtml = require('sanitize-html');
+const electron = require('electron');
+
+const analytics = electron.remote.getGlobal('analytics');
+const tingle = require('./vendor/tingle');
+const strings = require('./strings');
+
+// allowed html tags inside announcement
+const allowedTags = [
```

Comment:
lot of duplicate code in this file from menu. eventually would like that to be refactored to keep it [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

Reply:
I can move this to the strings file maybe?